README.md

```diff
@@ -235,6 +236,12 @@ gcp:
     projectid: "" # The GCP Project ID containing the Pub/Sub Topic
     topic: "" # The name of the Pub/Sub topic
   # minimumpriority: "debug" # minimum priority of event for using this output, order is emergency|alert|critical|error|warning|notice|informational|debug or "" (default)
+
+googleChat:
```

Comment:
Thank you for your quick reply.
Should this field be `googlechat` not `googleChat`?

Reply:
`googlechat` for respecting a global pattern

Reply:
Thank you, I will fix this now 👍

Reply:
@Issif Thank you for your reviews. I will this in https://github.com/falcosecurity/falcosidekick/pull/107/commits/0d31fe33b713bacbe673fc2cf1ec371508fa6b3c 🙇